Dockerfile

```diff
@@ -13,5 +14,6 @@ FROM openjdk:11-jdk-slim
 ARG path=/usr/app
 WORKDIR ${path}
 COPY --from=build ${path}/target/*.jar ${path}/app.jar
+COPY --from=build ${path}/*.json ${path}/internship-cloud/internship-project.json
```

Comment:
Why are you coping all json files if you need just one?

Reply:
I did this because we only have one json file. fixed